### PR TITLE
[WIP] some corrections

### PR DESCRIPTION
**Pull request by cordoval**
| Q | A |
| --- | --- |
| Bug Fix? | n |
| New Feature? | n |
| BC Breaks? | n |
| Deprecations? | n |
| Tests Pass? | n |
| Fixed Tickets |  |
| License | MIT |
| Doc PR |  |

 Sent using [Gush](https://github.com/gushphp/gush)

### CODE REVIEW

File: src/Buttercup/Protects/CorruptAggregateHistory.php [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 namespace Buttercup\Protects;
 
-use Exception;
-
-final class CorruptAggregateHistory extends Exception implements ButtercupProtectsException
+final class CorruptAggregateHistory extends \Exception implements ButtercupProtectsException
```
Comment (mathiasverraes) [inline src/Buttercup/Protects/CorruptAggregateHistory.php:5]: Why?

**Comment by mathiasverraes** [comment]
Appreciate it, but I can't see anything of value.

File: src/Buttercup/Protects/CorruptAggregateHistory.php (hunk 1 above)
Comment (cordoval) [inline src/Buttercup/Protects/CorruptAggregateHistory.php:5]: i think absolute namespaces make little sense when they are at the root namespace level no? less lines, easier to read and you don't end up with 10 use statements that are basic classes?

**Comment by cordoval** [comment]
it is ok, i do this as i am reading the code, sorry for the noise.

**Pull request closed without merging**